app/src/main/java/com/github/mik629/android/fundamentals/ui/moviedetails/FragmentMovieDetails.kt

```diff
@@ -1,90 +1,86 @@
 package com.github.mik629.android.fundamentals.ui.moviedetails
 
 import android.os.Bundle
-import android.view.LayoutInflater
 import android.view.View
-import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import com.bumptech.glide.load.resource.drawable.DrawableTransitionOptions
-import com.github.mik629.android.fundamentals.BuildConfig
-import com.github.mik629.android.fundamentals.GlideApp
+import androidx.fragment.app.viewModels
+import by.kirich1409.viewbindingdelegate.viewBinding
 import com.github.mik629.android.fundamentals.R
+import com.github.mik629.android.fundamentals.appComponent
 import com.github.mik629.android.fundamentals.databinding.FragmentMovieDetailsBinding
-import com.github.mik629.android.fundamentals.domain.model.MovieItem
+import com.github.mik629.android.fundamentals.domain.model.Movie
 import com.github.mik629.android.fundamentals.ui.global.ActorItemAdapter
+import com.github.mik629.android.fundamentals.ui.utils.buildGlideRequest
+import com.github.mik629.android.fundamentals.ui.utils.setRating
+import com.github.mik629.android.fundamentals.ui.utils.showSnackBar
+import javax.inject.Inject
 
-class FragmentMovieDetails : Fragment() {
-    private lateinit var binding: FragmentMovieDetailsBinding
+class FragmentMovieDetails : Fragment(R.layout.fragment_movie_details) {
+    private val binding by viewBinding(FragmentMovieDetailsBinding::bind)
+
+    @Inject
+    lateinit var movieDetailsViewModelFactory: MovieDetailsViewModelFactory.Factory
 
     private val glideRequest by lazy {
-        GlideApp.with(this)
-            .asDrawable()
-            .thumbnail(0.1f)
-            .placeholder(R.drawable.ic_image_loading)
-            .fallback(R.drawable.ic_broken_image)
-            .transition(DrawableTransitionOptions.withCrossFade())
-            .fitCenter()
+        buildGlideRequest(this)
+    } // stop storing link on this
```

Comment:
remove request

Reply:
I've changed it so it won't be storing link on this